src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/viewtakes/view/ViewTakesView.kt

```diff
@@ -43,194 +45,223 @@ class ViewTakesView : View() {
+                            // This take will not appear in the flow pane items
+                            if (it != null && selectedTakeProperty.value == null) {
+                                selectedTakeProperty.value = createTakeCard(it)
+                            } else if (it == null) selectedTakeProperty.value = null
+                        }
                     }
                 }
+
+                // Add the available takes flow pane
+                add(takesFlowPane)
             }
 
-            // Add the available takes flow pane
-            add(takesFlowPane)
-        }
+            // Record button?
+            val recordIcon = MaterialIconView(MaterialIcon.MIC_NONE, "25px")
+            recordButton = button("", recordIcon) {
+                anchorpaneConstraints {
+                    bottomAnchor = 25.0
+                    rightAnchor = 25.0
+                }
+                style {
+                    backgroundColor += c(Colors["base"])
+                    recordIcon.fill = c(Colors["primary"])
+                    backgroundRadius += box(100.0.px)
+                    borderRadius += box(100.0.px)
+                    prefHeight = 50.px
+                    prefWidth = 50.px
+                    effect = DropShadow(10.0, Color.GRAY)
+                }
 
-        // Record button?
-        val recordIcon = MaterialIconView(MaterialIcon.MIC_NONE, "25px")
-        recordButton = button("",recordIcon) {
-            anchorpaneConstraints {
-                bottomAnchor = 25.0
-                rightAnchor = 25.0
+                action {
+                    viewModel.recordTake()
+                }
             }
-            style {
-                backgroundColor += c(Colors["base"])
-                recordIcon.fill = c(Colors["primary"])
-                backgroundRadius += box(100.0.px)
-                borderRadius += box(100.0.px)
-                prefHeight = 50.px
-                prefWidth = 50.px
-                effect = DropShadow(10.0, Color.GRAY)
+
+            // Create drag shadow node and hide it initially
+            dragShadow = vbox {
+                draggingTakeProperty.onChange {
+                    clear()
+                    if (it != null) {
+                        add(it)
+                        show()
+                    } else {
+                        hide()
+                    }
+                }
+                hide()
+                addEventHandler(MouseEvent.MOUSE_DRAGGED, ::animateDrag)
+                addEventHandler(MouseEvent.MOUSE_RELEASED, ::completeDrag)
             }
         }
 
-        // Create drag shadow node and hide it initially
-        dragShadow = vbox {
-            draggingTakeProperty.onChange {
-                clear()
-                if (it != null) {
-                    add(it)
-                    show()
-                } else {
-                    hide()
+        stackpane {
```

Comment:
Since this is the same from the project page, maybe make this its own widget that can be reused for both views?